src/site/content/en/blog/defining-core-web-vitals-thresholds/index.md

```diff
@@ -0,0 +1,489 @@
+percentile value to be affected by the outliers.
+
+Given these goals are a bit at odds, after analysis, we concluded that the 75th
+percentile strikes a reasonable balance. By using the 75th percentile, we know
+that most visits to the site (3 of 4) experienced the target level of
+performance or better. Additionally, the 75th percentile value is less likely to
+be affected by outliers. Returning to our example, for a site with 100 visits,
+25 of those visits would need to report large outlier samples for the value at
+the 75th percentile to be affected by outliers. While 25 of 100 samples being
+outliers is possible, it is much less likely than for the 95th percentile case.
+
+## Largest Contentful Paint
+
+### Quality of experience
+
+1 second is often cited as the amount of time a user will wait before they begin
+to lose focus on a task. On closer inspection of relevant research, we found
+that 1 second is an approximation to describe a range of values, from roughly
+several hundred milliseconds to several seconds.
+
+Two commonly cited sources for the 1 second threshold are [Card et
+al](https://dl.acm.org/doi/10.1145/108844.108874) and
+[Miller](https://dl.acm.org/doi/10.1145/1476589.1476628). Card defines a
+1-second "immediate response" threshold, citing Newell's [Unified Theories of
+Cognition](https://dl.acm.org/doi/book/10.5555/86564). Newell explains immediate
+responses as "responses that must be made to some stimulus within _very
+approximately one second_ (that is, roughly from ~0.3sec to ~3sec)." This
+follows Newell's discussion on "real-time constraints on cognition", where it is
+noted that "interactions with the environment which evoke cognitive
+considerations take place on the order of seconds" which range from roughly 0.5
+to 2-3 seconds. Miller, another commonly cited source for the 1 second
+threshold, notes "tasks which humans can and will perform with machine
+communications will seriously change their character if response delays are
+greater than two seconds, with some possible extension of another second or so."
+
+Miller and Card's research describes the amount of time a user will wait before
+losing focus as a range, from roughly 0.3 to 3 seconds, which suggests our LCP
+"good" threshold should be in this range. Additionally, given that the existing
+First Contentful Paint "good" threshold is 1 second, and that the Largest
+Contentful Paint typically occurs after First Contentful Paint, we further
+constrain our range of candidate LCP thresholds, from 1 second to 3 seconds. To
+choose the threshold in this range that best meets our criteria, we look at the
+achievability of these candidate thresholds, below.
+
+### Achievability
+
+Using data from CrUX, we can determine the percentage of origins on the web that
+meet our candidate LCP "good" thresholds.
+
+**% of CrUX origins classified as "good" (for candidate LCP thresholds)**
+
+<div class="w-table-wrapper">
```

Comment:
This overflows on mobile. Not sure if it's worth the effort but we could use a media query to show a transposed table instead.

Reply:
cc: @robdodson (probably don't need to block this article since this style is used throughout the site)